### ISSUE TITLE
Additional test glyphs

### ISSUE THREAD

**Issue by rsheeter** [googlefonts/color-fonts#7]
To help test https://github.com/drott/freetype2-colr add:

*   Variations of the repeat modes and reference expectations
*   Variations of p2 for linear set and matrix transforms for radial and reference expectations.

**Comment by drott** [comment]
Clarification for 

> Variations of p2 for linear set and matrix transforms for radial and reference expectations.

What I mean is: Point p2 set for linear gradients, so that we can see and test the gradient rotated in various ways. Also rotated 90 degrees and more. Similarly radial: a transformation matrix set in various ways. Reference expectations should be documented / discussed.

**Comment by anthrotype** [comment]
in #8 I added six more glyphs to the samples font:

1) a skewed linear gradient with a P2 != P1:
    https://github.com/googlefonts/color-fonts/blob/master/font-srcs/samples/linear_transform.svg
2) a linear gradient with extend mode "reflect":
    https://github.com/googlefonts/color-fonts/blob/master/font-srcs/samples/linear_reflect.svg
3) a linear gradient with extend mode "repeat":
    https://github.com/googlefonts/color-fonts/blob/master/font-srcs/samples/linear_repeat.svg
4) a trasformed radial gradient:
    https://github.com/googlefonts/color-fonts/blob/master/font-srcs/samples/radial_transform.svg
5) a radial gradient with extend mode "reflect":
    https://github.com/googlefonts/color-fonts/blob/master/font-srcs/samples/radial_reflect.svg
6) a radial gradient with extend mode "repeat":
    https://github.com/googlefonts/color-fonts/blob/master/font-srcs/samples/radial_repeat.svg

The expected rendering should be the same as the original SVGs, of course.

We can add more SVGs to the `font-srcs/samples` directory and rerun `./update_samples.sh -t small_samples` shell script as we need.

**Comment by anthrotype** [comment]
I also added another glyph to the samples font which makes use of stop-opacity so you can test semi-transparent gradients:
https://github.com/googlefonts/color-fonts/blob/master/font-srcs/samples/gradient_stop_opacity.svg

**Comment by drott** [comment]
🥇  Thank you for adding these!

**Comment by rsheeter** [comment]
Is it tolerable if some scenarios are in a different font? - that's what I've done in #55 for now.

**Comment by drott** [comment]
Thanks for adding these and the infra to generate those, that's great. It'd be convenient to have them in one font, but for these 5, 6 we already have in the Skia font, I could back-port the SVG-source ones to this new test font? Might anyway be easier to tweak tests based on that? On the other hand - we have less of a reference if we don't have SVG source images?

**Comment by rsheeter** [comment]
> we have less of a reference if we don't have SVG source images?

Agreed but there are cases you can't express in svg (sweep/conical gradient) and cases we won't currently emit. 

I suppose one thing I could do is draw all the geometry in svg and only script the COLR parts. My gut says that makes my life more difficult for minimal benefit but maybe that's not true, being able to "see" the input is worth the effort (even if things like the sweep or composition setup aren't present). Maybe I'll merge #55 for now and think a little more about that when adding more examples.
